acoustid/data/fingerprint.py

```
@@ -215,7 +216,7 @@ def _search_via_fpstore(
     def _search_directly(
         self, fp: List[int], length: int, max_results: Optional[int] = None
     ) -> List[FingerprintMatch]:
-        conditions = []
+        conditions: List[ColumnElement[bool]] = []
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)